core/tee/tee_svc.c

```diff
@@ -815,9 +831,15 @@ static TEE_Result tee_svc_update_out_param(
 {
 	size_t n;
```

Comment:
While you're at it, please initialize this too.

Reply:
Okay, updated